trame_code/widgets/code.py

```diff
@@ -21,8 +21,8 @@ class Editor(HtmlElement):
 
     Properties:
 
+    :param v_model:
```

Comment:
v_model is a directive, that is not part of the properties... Directive apply to any components.